esx_service/vmdk_ops.py

```diff
@@ -854,11 +854,14 @@ def findDeviceByPath(vmdk_path, vm):
         # to match with the given volume name.
         # Filename format is as follows:
         #   "[<datastore name>] <parent-directory>/tenant/<vmdk-descriptor-name>"
-        logging.debug("d.backing.fileName %s", d.backing.fileName)
-        backing_disk = d.backing.fileName.split(" ")[1]
+        # Assume there is one space between these two parts "[<datastore name>]" and
```

Comment:
This comment is not very clear to me. I'd suggest to give an example in the comment so that it's easier to understand the logic of ....rsplit(" ", 1)[0]